Learning/README.md

```diff
@@ -18,14 +18,75 @@ The most important (class) files are mentioned in this table.
 
 Install the required pip packages and the mentioned packages from the README in the main folder.
 
-### data_collection.py
+To train a supervised learning policy we need training data. Experiments showed that running 90-150 gardens with 200 plants from 10 plant types, 
+0.1 _PERCENT_NON_PLANT_CENTERS_ and 100 garden days deliver a reasonable amount of training data. 
 
-To run the simulator and collect data:
+We can generate the data for one garden with `data_collection.py` or for a batch with `collect.py`.
+After having generating data, next step is to normalize the generated data with `moments.py`.
+To adjust the network architecture update the code in `net.py`. Before training, additional adjustments may be needed to be made in `trainer.py`, e.g. set GPU usage or amount of workers for loading training data.
+Before training, the normalised data generated with `moments.py` needs to be connected to the generated data. Last but not least, training is happening with `train.py`.
```

Comment:
Not sure what "connected to the generated data means".  Can remove.

Reply:
Also change "Last but not least, training is happening with train.py" to something like "Last but not least, you can train your network with train.py"